dockerfiles/che/Dockerfile.centos

```diff
@@ -8,17 +8,16 @@
 #   Dharmit Shah  - Initial implementation
 #   Mario Loriedo - Improvements
 #
-# To build it, run in the current folder after executing build.sh:
-#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
+# To build it, in current folder:
+#  cp Dockerfile.centos Dockerfile
+#  ./build.sh
 #
 # To run it:
-#  docker run --net=host \
-#             --name che \
-#             -v /var/run/docker.sock:/var/run/docker.sock \
-#             -v /home/user/che/lib:/home/user/che/lib-copy \
-#             -v /home/user/che/workspaces:/home/user/che/workspaces \
-#             -v /home/user/che/storage:/home/user/che/storage \
-#             registry.centos.org/eclipse/che-server
+#  docker run -d -p 8080:8080 \
+#            --name che \
+#            -v /var/run/docker.sock:/var/run/docker.sock \
+#            -v ~/.che/workspaces:/data \
+#            eclipse/che-server:nightly
```

Comment:
@JamesDrummond I like `~/.che/` more because I can just copy paste and it works. I don't need an extra step of editing the command. But I understand you want to make it clear that the path is configurable. I just wanted to be sure that you understand why I prefer `~/.che/` (and potentially other users would prefer it too). If you still think that setting `<LOCAL_PATH>` is a better choice I will change that. 

What is the issue with nightly? I understand you want to change that but I don't understand why. I'm asking because in the past I've fixed an issue with **non** nightly tags and I ran into another issue recently. So when I build it I usually use the default tag (nightly) to improve my chance to build/run successfully.